webpack.config.js

```diff
@@ -85,6 +85,17 @@ module.exports = (env, argv) => {
             }
         },
         optimization: {
+            minimizer: [
+                new TerserPlugin({
+                    sourceMap: true,
+                    parallel: true,
+                    cache: './.build_cache/terser',
```

Comment:
Two things matter here: (1) the cache folder, (2) source mapping in the minimizer, this allows the devtool plugin to override which chunk will get mapped (vs. `devtools` prop)